rotate.go

```diff
@@ -66,6 +66,14 @@ type Config struct {
 	// EventReceiver receives events during the four-step password rotation process.
 	// If none is provided, NullEventReceiver is used. See EventReceiver for more details.
 	EventReceiver EventReceiver
+
+	// ReplicationWaitRetries governs how many times password rotation lambda
+	// will retry waiting for secret replication to secondary region to become in sync
+	ReplicationWaitRetries int
+
+	// ReplicationWaitDuration governs the duration password rotation lambda will wait
+	// between retries of checking for secret replication status to secondary regions
+	ReplicationWaitDuration time.Duration
```

Comment:
Let's make this simpler with a single var: `ReplicationWait time.Duration`. Default... I don't know, 30s?

Then it's easier to document and communicate to user: how long do you want to wait for the secret to replicate?

This also makes the internal loop a little simpler like:

```go
for time.Now().Sub(startTime) < ReplicationWait {
    // Try
    time.Sleep(500 * time.Millisecond)
}
```

It'll also make the code resume faster: 500ms vs 5s.  500ms is fast enough that maybe a human won't notice, but not so fast that'll overload the AWS API rate limiting.

Reply:
Updated with a default 10 second timeout.

Reply:
Updated